assets/README.md

```diff
@@ -0,0 +1,29 @@
+Kuma Assets
+===========
+This folder contains front-end assets for developer.mozilla.org.
+
+In production, they are collected to ``/static``, processed to add
+a hash of the contents to the name, and served to users.
+In development, some may be served from this folder, to speed up
+development. See
+[Building, collecting, and serving assets](https://kuma.readthedocs.io/en/latest/assets.html)
+for the details.
+
+Here are the subfolders:
+
+* **build**: (TODO) Output of ``gulp`` and ``make compilejsi18n``, copied to
+  ``static/build``.
+* **ckeditor**: (TODO) The source and build folders for CKEditor.
+  ``ckeditor/build`` is copied to ``static/js/libs/ckeditor/build``.
+* **dist**: (TODO) Output of the ``webpac``k process, copied to ``static/dist``
+* **layout-tests**: (TODO) Some statically-served pages for speed testing,
```

Comment:
I wonder if we still need to keep these. @atopal?

Reply:
No, you can remove them. I can still access the tests I have run in in Q4, and I can't use those pages for current testing anyway, because the hash added to file names for cache busting means those files are 404ing now.